packages/react/stories/DataTable.stories.tsx

```diff
@@ -33,34 +34,42 @@ export const Default = () => {
     {
       header: 'First Name',
       accessorKey: 'firstName',
+      footer: (props) => props.column.id,
     },
     {
       header: 'Last Name',
       accessorKey: 'lastName',
+      footer: (props) => props.column.id,
     },
     {
       header: 'Gender',
       accessorKey: 'gender',
+      footer: (props) => props.column.id,
     },
     {
       header: 'Age',
       accessorKey: 'age',
+      footer: (props) => props.column.id,
     },
     {
       header: 'Address',
       accessorKey: 'address',
+      footer: (props) => props.column.id,
     },
     {
       header: 'City',
       accessorKey: 'city',
+      footer: (props) => props.column.id,
     },
     {
       header: 'Subscribed',
       accessorKey: 'isSubscribed',
+      footer: (props) => props.column.id,
```

Comment:
I don't see `footer` props is used anywhere

Reply:
removing these

Reply:
they're still there